src/fromager/wheels.py

```diff
@@ -15,6 +15,9 @@ def build_wheel(ctx, req, sdist_root_dir, build_env):
     if not builder:
         builder = default_build_wheel
     extra_environ = overrides.extra_environ_for_pkg(ctx.envs_dir, req.name, ctx.variant)
+    # TODO: refactor?
+    # Build Rust without network access
+    extra_environ["CARGO_NET_OFFLINE"] = "true"
```

Comment:
I wonder if this should go in `default_build_wheel`? Maybe it's better to do it here so any overrides also use the same setting without having to remember to add it themselves.